nova3/engines/piratebay.py

```diff
@@ -1,4 +1,4 @@
-# VERSION: 3.3
+# VERSION: 3.4
```

Comment:
You should also bump version in [versions.txt](https://github.com/qbittorrent/search-plugins/blob/master/nova3/engines/versions.txt).